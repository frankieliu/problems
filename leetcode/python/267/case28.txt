answer: [
"aabbccccbbaa"
"aabcbccbcbaa"
"aabccbbccbaa"
"aacbbccbbcaa"
"aacbcbbcbcaa"
"aaccbbbbccaa"
"ababccccbaba"
"abacbccbcaba"
"abaccbbccaba"
"abbaccccabba"
"abbcaccacbba"
"abbccaaccbba"
"abcabccbacba"
"abcacbbcacba"
"abcbaccabcba"
"abcbcaacbcba"
"abccabbaccba"
"abccbaabccba"
"acabbccbbaca"
"acabcbbcbaca"
"acacbbbbcaca"
"acbabccbabca"
"acbacbbcabca"
"acbbaccabbca"
"acbbcaacbbca"
"acbcabbacbca"
"acbcbaabcbca"
"accabbbbacca"
"accbabbabcca"
"accbabbabcca"
"accbbaabbcca"
"accbbaabbcca"
"baabccccbaab"
"baacbccbcaab"
"baaccbbccaab"
"babaccccabab"
"babcaccacbab"
"babccaaccbab"
"bacabccbacab"
"bacacbbcacab"
"bacbaccabcab"
"bacbcaacbcab"
"baccabbaccab"
"baccbaabccab"
"bbaaccccaabb"
"bbacaccacabb"
"bbaccaaccabb"
"bbcaaccaacbb"
"bbcacaacacbb"
"bbccaaaaccbb"
"bcaabccbaacb"
"bcaabccbaacb"
"bcaacbbcaacb"
"bcaacbbcaacb"
"bcabaccabacb"
"bcabaccabacb"
"bcabcaacbacb"
"bcabcaacbacb"
"bcacabbacacb"
"bcacabbacacb"
"bcacbaabcacb"
"bcacbaabcacb"
"bcbaaccaabcb"
"bcbacaacabcb"
"bcbcaaaacbcb"
"bccaabbaaccb"
"bccabaabaccb"
"bccbaaaabccb"
"caabbccbbaac"
"caabbccbbaac"
"caabcbbcbaac"
"caabcbbcbaac"
"caacbbbbcaac"
"caacbbbbcaac"
"cababccbabac"
"cababccbabac"
"cabacbbcabac"
"cabacbbcabac"
"cabbaccabbac"
"cabbaccabbac"
"cabbcaacbbac"
"cabbcaacbbac"
"cabcabbacbac"
"cabcabbacbac"
"cabcbaabcbac"
"cabcbaabcbac"
"cacabbbbacac"
"cacabbbbacac"
"cacbabbabcac"
"cacbabbabcac"
"cacbabbabcac"
"cacbabbabcac"
"cacbbaabbcac"
"cacbbaabbcac"
"cacbbaabbcac"
"cacbbaabbcac"
"cbaabccbaabc"
"cbaabccbaabc"
"cbaacbbcaabc"
"cbaacbbcaabc"
"cbabaccababc"
"cbabaccababc"
"cbabcaacbabc"
"cbabcaacbabc"
"cbacabbacabc"
"cbacabbacabc"
"cbacbaabcabc"
"cbacbaabcabc"
"cbbaaccaabbc"
"cbbacaacabbc"
"cbbcaaaacbbc"
"cbcaabbaacbc"
"cbcabaabacbc"
"cbcbaaaabcbc"
"ccaabbbbaacc"
"ccababbabacc"
"ccabbaabbacc"
"ccbaabbaabcc"
"ccbabaababcc"
"ccbbaaaabbcc"
]
