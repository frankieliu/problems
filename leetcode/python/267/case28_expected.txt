expected_answer: [
"aabbccccbbaa"
"aabcbccbcbaa"
"aabccbbccbaa"
"aacbbccbbcaa"
"aacbcbbcbcaa"
"aaccbbbbccaa"
"ababccccbaba"
"abacbccbcaba"
"abaccbbccaba"
"abbaccccabba"
"abbcaccacbba"
"abbccaaccbba"
"abcabccbacba"
"abcacbbcacba"
"abcbaccabcba"
"abcbcaacbcba"
"abccabbaccba"
"abccbaabccba"
"acabbccbbaca"
"acabcbbcbaca"
"acacbbbbcaca"
"acbabccbabca"
"acbacbbcabca"
"acbbaccabbca"
"acbbcaacbbca"
"acbcabbacbca"
"acbcbaabcbca"
"accabbbbacca"
"accbabbabcca"
"accbbaabbcca"
"baabccccbaab"
"baacbccbcaab"
"baaccbbccaab"
"babaccccabab"
"babcaccacbab"
"babccaaccbab"
"bacabccbacab"
"bacacbbcacab"
"bacbaccabcab"
"bacbcaacbcab"
"baccabbaccab"
"baccbaabccab"
"bbaaccccaabb"
"bbacaccacabb"
"bbaccaaccabb"
"bbcaaccaacbb"
"bbcacaacacbb"
"bbccaaaaccbb"
"bcaabccbaacb"
"bcaacbbcaacb"
"bcabaccabacb"
"bcabcaacbacb"
"bcacabbacacb"
"bcacbaabcacb"
"bcbaaccaabcb"
"bcbacaacabcb"
"bcbcaaaacbcb"
"bccaabbaaccb"
"bccabaabaccb"
"bccbaaaabccb"
"caabbccbbaac"
"caabcbbcbaac"
"caacbbbbcaac"
"cababccbabac"
"cabacbbcabac"
"cabbaccabbac"
"cabbcaacbbac"
"cabcabbacbac"
"cabcbaabcbac"
"cacabbbbacac"
"cacbabbabcac"
"cacbbaabbcac"
"cbaabccbaabc"
"cbaacbbcaabc"
"cbabaccababc"
"cbabcaacbabc"
"cbacabbacabc"
"cbacbaabcabc"
"cbbaaccaabbc"
"cbbacaacabbc"
"cbbcaaaacbbc"
"cbcaabbaacbc"
"cbcabaabacbc"
"cbcbaaaabcbc"
"ccaabbbbaacc"
"ccababbabacc"
"ccabbaabbacc"
"ccbaabbaabcc"
"ccbabaababcc"
"ccbbaaaabbcc"
]
